packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -61,3 +94,49 @@ export const useAvatarGroupItemStyles_unstable = (state: AvatarGroupItemState):
 
   return state;
 };
+
+/**
+ * Hook for getting the className for the children of AvatarGroup. This hook will provide the spacing and outlines
+ * needed for each layout.
+ */
+export const useLayoutClassName = (layout: AvatarGroupProps['layout'], size?: AvatarSizes): string => {
```

Comment:
`size` shouldn't be optional here. It looks like this is probably because you're getting the size from the avatar prop, but perhaps you should get it from the group context instead?

(And maybe put "`groupSize`" and "`groupLayout`" on the context, so you're not duplicating calls to `useContextSelector`)

```suggestion
export const useLayoutClassName = (layout: AvatarGroupProps['layout'], size: AvatarSizes): string => {
```